### PR TITLE
More Synopses

### PR DESCRIPTION
**Pull request by RAMilewski**

### CODE REVIEW

File: masks3d.scad [hunk 1]
```diff
@@ -225,10 +237,13 @@ module rounding_edge_mask(l, r, r1, r2, d, d1, d2, excess=0.1, anchor=CENTER, sp
 
 
 // Module: rounding_corner_mask()
+// Synopsis: Creates a shape to round 90° corners.
+// Topics: Masking, Rounding, Shapes (3D)
+// See Also: angled_rounding_corner_mask(), rounding_edge_mask(), rounding_angled_edge_mask()
```
Comment (revarbat) [inline masks3d.scad:242]: Error: !! ERROR at masks3d.scad:242: Invalid Link 'angled_rounding_corner_mask()'

File: masks3d.scad [hunk 2]
```diff
@@ -486,10 +513,13 @@ module rounding_hole_mask(r, rounding, excess=0.1, d, anchor=CENTER, spin=0, ori
 // Section: Teardrop Masking
 
 // Module: teardrop_edge_mask()
+// Synopsis: Creates a shape to round a 90° edge but limit the angle of overhang.
+// Topics: Masking, Rounding, Shapes (3D), FDM Optimized
+// See Also: teardrop_corner_mask
```
Comment (revarbat) [inline masks3d.scad:518]: Error: !! ERROR at masks3d.scad:518: Invalid Link 'teardrop_corner_mask'

I think it needs the ending () parens.

File: masks3d.scad [hunk 3]
```diff
@@ -525,6 +555,9 @@ module teardrop_edge_mask(l, r, angle=45, excess=0.1, d, anchor=CTR, spin=0, ori
 
 
 // Module: teardrop_corner_mask()
+// Synopsis: Creates a shape to round a 90° corner but limit the angle of overhang.
+// Topics: Masking, Rounding, Shapes (3D), FDM Optimized
+// See Also: teardrop_edge_mask
```
Comment (revarbat) [inline masks3d.scad:560]: Error: !! ERROR at masks3d.scad:560: Invalid Link 'teardrop_edge_mask'

Again, I think it needs the ending () parens.

**Comment by RAMilewski** [comment]
Is there a way I can run the CheckDocs test here?  Is that run_tests.sh in the scripts folder?

**Comment by RAMilewski** [comment]
OK.  I think I see how this works now.

**Comment by revarbat** [comment]
CheckDocs just runs `openscad-docsgen` with an argument that makes animations only test the first frame.

**Pull request merged**